src/conversations/Conversation.ts

```diff
@@ -13,11 +13,20 @@ import Message from '../Message'
  */
 export default class Conversation {
   peerAddress: string
+  context?: InvitationContext
   private client: Client
+  private topics: string[]
 
-  constructor(client: Client, address: string) {
+  constructor(
+    client: Client,
+    address: string,
+    topics: string[],
+    context?: InvitationContext
+  ) {
     this.peerAddress = address
     this.client = client
+    this.topics = topics
```

Comment:
Conversations have the topics explicitly defined now. Can be multiple topics.

Haven't gotten around to rewriting the message reading/writing functions to use the topic array yet, but that should be pretty straightforward.

Reply:
So this is for stitching conversations from multiple invitations? Do we want to go there at this point?

Reply:
I've been really designing this around stitching from 1 invitation and one intro message, which is a good place to start. Considering we don't have topic rotation, that will be the only use-case we support to start